src/main/java/com/sericulture/authentication/controller/UserInfoController.java

```diff
@@ -58,11 +58,11 @@ public AuthApiResponse checkToken() {
         return authApiResponse;
     }
 
-    @PostMapping("/register")
+  /*  @PostMapping("/register")
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Block comment has incorrect indentation level 2, expected is 4, indentation should be the same level as line 67.